pkgs/data/fonts/noto-fonts/default.nix

```diff
@@ -11,32 +11,44 @@
 , imagemagick
 , zopfli
 , buildPackages
+, fontconfig
+, python3
 }:
 
 let
-  mkNoto = { pname, weights }:
+  noto-pkg =
     stdenvNoCC.mkDerivation {
-      inherit pname;
-      version = "2020-01-23";
+      pname = "noto-fonts";
+      version = "unstable-2022-04-25";
 
       src = fetchFromGitHub {
         owner = "googlefonts";
         repo = "noto-fonts";
-        rev = "f4726a2ec36169abd02a6d8abe67c8ff0236f6d8";
-        sha256 = "0zc1r7zph62qmvzxqfflsprazjf6x1qnwc2ma27kyzh6v36gaykw";
+        rev = "a19de47f845dbd4c61b884c7ff90ce993555d05d";
+        sparseCheckout = ''
+          unhinted/otf
+          unhinted/variable-ttf
+        '';
+        hash = "sha256-7Jzo7402CiOcn2q++zbAzn2tP/y2x7QJuwjcP1NoyLQ=";
       };
 
+      analyzeScan = ./analyze_scan.py;
```

Comment:
Can you add a comment what this does roughly?

Reply:
It reads all the font information from fc-scan and figures out which fonts are included in the variable fonts and which aren't and need the OTF version.

Reply:
There probably is a way to do it with shell scripting but my head hurt thinking about it.